.github/workflows/python-tests/action.yml

```
@@ -0,0 +1,35 @@
+name: "Setup environment and run python tests in tox"
```

```suggestion
name: "Setup Environment and run Python Tests in Tox"
```

You can ignore this if you don't want this to follow same format as the other actions.